public/aboutus.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en-US">
+<head>
+  <!-- v8299 -->
+  <title>About SG Kampung &ndash; SGKampung</title>
+
+  <meta name="csrf-param" content="authenticity_token" />
+<meta name="csrf-token" content="QjMUguAV6ChhPjfBmSieN4HvizBcLU3hyEbvFl2WT7UHgDUMwGi6tIZr3vrJAzUZm3WLtw3P8FAW5cmUvpGRMg==" />
+  <meta name="description" content="What is SG Kampung?  SG Kampung is the free social network being used in thousands of Constituents across Singapore, helping residents to get the best out..." /><meta property="og:image" content="http://p4.zdassets.com/hc/settings_assets/571017/200048839/WeuCIW71z70Mkx7GbJTOFg-SG Kampung-logo-transparent-170.png" />
+<meta property="og:type" content="website" />
+<meta property="og:site_name" content="SG Kampung" />
+<meta property="og:title" content="About SG Kampung" />
+<meta property="og:description" content="What is SG Kampung?
+SG Kampung is the free social network being used in thousands of Constituents across Singapore, helping residents to get the best out of their local area.
+There are many reasons to join SG Kampung:..." />
+<meta property="og:url" content="http://help.SG Kampung/hc/en-us/articles/202364115-About-SG Kampung" />
+<link href="https://help.SG Kampung/hc/en-us/articles/202364115-About-SG Kampung" rel="canonical" />
```

Comment:
its a 404, the correct content if a functionality is required, has not been put in place.  If no functionality relates, will delete line 16